sdk-libs/macros/docs/features/rentfree-features.md

```
@@ -0,0 +1,499 @@
+    CreateProfileCompressed,  // Compressed variant
+    UpdateProfile,
+    CompressProfile,          // Compression instruction
+    DecompressProfile,        // Decompression instruction
+}
+```
+
+---
+
+## Execution Flow Comparison
+
+### Anchor Standard Flow
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)